### ISSUE TITLE
Pre-release suffix breaks Sencha package functionality

### ISSUE THREAD

**Issue by bentrm** [geoext/geoext2#374]
Haven't used this in a while but just noticed, that the "-dev" version suffix in `package.json` of upstream breaks the Sencha Cmd tool. It quits with the following:

```
Sencha Cmd v6.0.2.14
[INF] Processing Build Descriptor : default

[ERR] For input string: "1-dev"
[ERR] A crash log is available in the file "sencha-error-20160109.log" please contact Sencha Support and share this file with us.
```

Removing the suffix fixes this. Is this acceptable or should the master branch be as usable as release versions with the toolchain as well?

**Comment by marcjansen** [comment]
I don't want to break the build. Let's remove the 'dev'

For gx3 I've noticed this behaviour, for gx2 I forgot about that.

**Comment by bentrm** [comment]
Fixed with #375.
